app/main.py

```diff
@@ -98,17 +122,83 @@ async def swagger_ui_redirect():
 for router in routers_to_include:
     app.include_router(router)
 
+EMPTY_HEART_PATH = "media/empty_heart.png"
+FULL_HEART_PATH = "media/full_heart.png"
+
 
-# TODO: I add the quote day to the home page
-# until the relevant calendar view will be developed.
 @app.get("/", include_in_schema=False)
 @logger.catch()
-async def home(request: Request, db: Session = Depends(get_db)):
+async def home(
+    request: Request,
+    db: Session = Depends(get_db),
+    user: User = Depends(current_user),
+):
+    """Home page for the website."""
     quote = daily_quotes.get_quote_of_day(db)
-    return templates.TemplateResponse("index.html", {
-        "request": request,
-        "quote": quote,
-    })
+    user_quotes = daily_quotes.get_quotes(db, user.user_id)
+    for user_quote in user_quotes:
+        if user_quote.id == quote.id:
+            quote.is_favorite = True
```

Comment:
It would be better to add this to the query (that's the rule of databases, to easily and efficiently select and filter things)